api/status.proto

```diff
@@ -0,0 +1,92 @@
+syntax = "proto3";
+
+option go_package = "github.com/clyso/ceph-api/api/ceph;pb";
+
+package ceph;
+
+import "google/protobuf/empty.proto";
+import "google/protobuf/struct.proto";
+
+service Status {
+  // command: ceph status
+  rpc GetCephStatus (google.protobuf.Empty) returns (GetCephStatusResponse) {}
+}
+
+message GetCephStatusResponse {
+  string fsid = 1;
+  CephStatusHealth health = 2;
+  int32 election_epoch = 3;
+  repeated int32 quorum = 4;
+  repeated string quorum_names = 5;
+  int32 quorum_age = 6;
```

Comment:
so in this context epoch and age is not time but some internal ceph sequence, right? And we cannot convert it to timestamps?

Reply:
election epoch is a sequence used to track the number of elections that have occured, and quorum age is the number of seconds passed since the quroum has been stable both can be represented as integer, not sure weather quorums can be stable for more than the upper limit of int32 in go....

Reply:
to be on the safe side we can just upgrade all int32 variables to int64?